docs/DeveloperGuide.md

```diff
@@ -271,25 +271,31 @@ _{Explain here how the data archiving feature will be implemented}_
+|----------|-----------------------------------------------|-----------------------------------------------------|------------------------------------------------------------------------|
+| `* * *`  | Student                                       | see usage instructions                              | refer to instructions when I forget how to use the App                 |
+| `* * *`  | Student                                       | add a new contact                                   |                                                                        |
+| `* * *`  | Student                                       | delete a contact                                    | remove a contact that I no longer need                                 |
+| `* * *`  | Student                                       | view a person's contact details                     | contact the person(TA/Professor) to seek help for my tutorials         |
+| `* * *`  | Student                                       | find a contact by name                              | locate details of persons without having to go through the entire list |
+| `* * *`  | Student                                       | filter my contacts by tag                           | find the persons that is related to the tag quickly                    |
+| `* * *`  | Student                                       | edit a contact                                      | update the contact details of my contacts when they change             |
+| `* * *`  | Student                                       | assign modules to my contacts                       | know which of my contacts are in charge of which modules               |
+| `* * *`  | Student                                       | filter my contacts by module                        | find all the relevant contacts of a module I am taking                 |
+| `* *`    | Student                                       | undo my last command                                | reverse my actions if i made a wrong change to ModCheck                |
+| `* *`    | Student                                       | be able to set certain fields as 'unknown'          | add contacts that I may not know all the details of                    |
+| `* *`    | Student                                       | hide private contact details                        | minimize chance of someone else seeing them by accident                |
+| `* *`    | Student                                       | delete all contacts                                 | remove all contacts for a fresh start                                  |
+| `*`      | Student with many persons in the address book | sort persons by name                                | locate a person easily                                                 |
```

Comment:
Small change to prevent references to `Address book`

```suggestion
| `*`      | Student with many contacts | sort contacts by name                                | locate a contact easily                                                 |
```